docs/cases/cases-overview.asciidoc

```diff
@@ -118,9 +118,9 @@ IMPORTANT: Before importing Lens visualizations, Timelines, or alerts into a spa
 [float]
 [[cases-export]]
 === Export a case
-Use the export option to move cases between different Kibana instances. When you export a case, the following data is exported to a newline-delimited JSON (NDJSON) file: case details, user actions, text string comments, case alerts, and lens visualizations.
+Use the export option to move cases between different Kibana instances. When you export a case, the following data is exported to a newline-delimited JSON (`.ndjson`) file: case details, user actions, text string comments, case alerts, and lens visualizations (which are exported as JSON blobs).
 
-NOTE: Exported alerts contain metadata that allow the case to reference the appropriate alerts after they're imported. Lens visualizations are exported as JSON blobs.
+IMPORTANT: Alerts attached to a case are exported as references containing alert metadata. When a <<cases-import, case is imported>>, it uses these references to re-connect to the appropriate alerts. However, alert references only resolve if the correct alert documents are available in the environment. If they're not, alert details will not load when you access them from the imported case. Links to the rules that produced the alert are treated the same way; they only resolve if the rule exists in the environment. 
```

Comment:
> Links to the rules that produced the alert are treated the same way; they only resolve if the rule exists in the environment.

So there's a slight nuance here with regards to links to Rule Details as we do have support for showing 'Deleted Rules', which this would classify as (even though it never existed). For this functionality (most recently updated in https://github.com/elastic/kibana/pull/120053), if a Rule doesn't exist, we'll try to show its details from the source Alert (by means of the RuleId which should stay static between Rule imports/exports). So if they re-import the associated Alert, links to Rule Details should still function, it'll just have a badge next to the name saying it was deleted.

Reply:
Thank you so much @nastasha-solomon and @spong! I wasn't aware of such functionality. Today I learned something 🙂.